lib/puppet/provider/oneview_ethernet_network/c7000.rb

```diff
@@ -86,6 +86,13 @@ def bulk_create_check
   # Bulk deletes networks if there is @data['networkUris']
   def bulk_delete_check
     if @data['networkUris']
+      list = []
+      @data['networkUris'].each do |item|
+        net = OneviewSDK.resource_named(:EthernetNetwork, api_version, resource_variant).find_by(@client, name: item)
```

Comment:
What happens if user wants to provide URI directly instead of name..? Are we handling that scenario..?
Refer this https://github.com/HewlettPackard/oneview-puppet/blob/master/lib/puppet/provider/oneview_logical_interconnect_group/c7000.rb#L113

Reply:
Done